truss/tests/test_model_inference.py

```diff
@@ -137,6 +137,58 @@ def _test_invocations(expected_code):
         assert not _test_invocations(expected_code=200)
 
 
+@pytest.mark.integration
+def test_streaming_truss():
+    with ensure_kill_all():
+        truss_root = Path(__file__).parent.parent.parent.resolve() / "truss"
+        truss_dir = truss_root / "test_data" / "test_streaming_truss"
+        tr = TrussHandle(truss_dir)
+
+        _ = tr.docker_run(local_port=8090, detach=True, wait_for_server_ready=False)
+
+        SERVER_WARMUP_TIME = 3
+
+        truss_server_addr = "http://localhost:8090"
+
+        time.sleep(SERVER_WARMUP_TIME)
+
+        def _test_liveness_probe(expected_code):
+            live = requests.get(f"{truss_server_addr}/")
+            assert live.status_code == expected_code
+
+        def _test_readiness_probe(expected_code):
+            ready = requests.get(f"{truss_server_addr}/v1/models/model")
+            assert ready.status_code == expected_code
+
+        def _test_ping(expected_code):
+            ping = requests.get(f"{truss_server_addr}/ping")
+            assert ping.status_code == expected_code
+
+        _test_liveness_probe(200)
+        _test_readiness_probe(200)
+        _test_ping(200)
+
+        invocations_response = requests.post(
```

Comment:
s/invocations/predict/g for the rest of this test. invocations is a Sagemaker endpoint and we don't know if there are other intricacies to get Sagemaker to stream so this test can be a false positive in that case.